backend/globaleaks/db/sqlite.sql

```diff
@@ -429,4 +429,18 @@ CREATE TABLE archivedschema (
     schema BLOB NOT NULL,
     UNIQUE (hash, type, language),
     PRIMARY KEY (id)
-)
+);
+
+CREATE TABLE log (
+    id INTEGER NOT NULL,
+    code TEXT NOT NULL,
+    args VARCHAR,
+    log_date TEXT NOT NULL,
+    subject TEXT NOT NULL,
+    log_message TEXT NOT NULL,
+    log_level INTEGER,
+    mail INTEGER,
```

Comment:
on purpose.. mail is a boolean (when is False value 0, is the 0 considered NULL ?), and log_level can be also 0. is appropriate therefore keep the vars without constraint ?